topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -914,7 +914,7 @@ To compare samples or gene expressions, the gene counts need to be normalized. W
 >    **Total reads** | 35 | 45 | 106
 >    **Scaling factor** | 3.5 | 4.5 | 10.6
 >
->    *Because of the small values in the example, we are scoring using a factor of 10.*
+>    *Because of the small values in the example, we are scoring using a factor of 10.* <!-- confusing -->
 >
```

Comment:
I think the scoring by a factor of 10 rather than 10^6 makes the explanation more confusing than it needs to be.